src/root/start.js

```diff
@@ -21,10 +21,10 @@ window.c.root.Start = ((m, c, h, models, I18n) => {
                 uservm = filters({
                     id: 'eq'
                 }),
-                loader = m.postgrest.loaderWithToken,
+                loader = _.partial(m.postgrest.loader, _, m.postgrest.request),
```

Comment:
we have the same code in ```src/root/projects-home.js```
I've have a proposal to change this interface in mithril.postgrest here: https://github.com/catarse/mithril.postgrest/pull/23

If you think that's a better solution you can merge and release a new mithril.postgrest version to refactor this ;)
Just remember to bump the minor version and refactor other calls in catarse.js since we are breaking the interface.